test/plug_context_test.exs

```diff
@@ -38,6 +42,31 @@ defmodule Sentry.PlugContextTest do
            } = Sentry.Context.get_all()
   end
 
+  test "sets request context with real client ip if request is forwarded" do
+    Sentry.PlugContext.call(
+      conn(:get, "/test?hello=world") |> add_x_forwarded_for("10.0.0.1"),
```

Comment:
Does it make sense to add a test with more than one IP address?

Reply:
Yes, it will be more explicit.

You may have multiple values for this header. In that case, the first one is the client address and the next ones the succession of proxies until the final server.

`X-Forwarded-For: <client>, <proxy1>, <proxy2>`